export/registration_test.go

```diff
@@ -11,26 +11,30 @@ import "testing"
 func TestRegistrationValid(t *testing.T) {
 	var tests = []struct {
 		name        string
+		regName     string
 		compression string
 		format      string
 		destination string
 		encryption  string
 		valid       bool
 	}{
-		{"empty", "", "", "", "", false},
-		{"valid", CompZip, FormatJSON, DestMQTT, EncAes, true},
-		{"defaultCompression", "", FormatJSON, DestMQTT, EncAes, true},
-		{"defaultEncryption", CompZip, FormatJSON, DestMQTT, "", true},
-		{"wrongCompresion", "INVALID", FormatJSON, DestMQTT, EncAes, false},
-		{"wrongFormat", CompZip, "INVALID", DestMQTT, EncAes, false},
-		{"wrongDestination", CompZip, FormatJSON, "INVALID", EncAes, false},
-		{"wrongEncryption", CompZip, FormatJSON, DestMQTT, "INVALID", false},
+		{"empty", "reg", "", "", "", "", false},
```

Comment:
Change the name to empty string in the "empty" case. If not the case with an empty name is not tested.

Reply:
I also added case "withoutName" which checks the case of empty name, so already is tested. But if you want, I can remove this new case and modify "empty" case

Reply:
I missed your new entry. Sorry.
But I still think that is better to have the "empty" entry with a default empty registration.

And it will also launch new jenkins jobs.

Reply:
I think my comment was not clear. It is better to keep both in the tests. Just remove the name in the empty registration.